tensorboard/plugins/graph/tf_graph/tf-graph.ts

```diff
@@ -173,6 +176,7 @@ class TfGraph extends LegacyElementMixin(PolymerElement) {
     (this.$$('tf-graph-scene') as any).panToNode(nodeName);
   }
   @observe(
+    'extractNodes',
```

Comment:
When manually testing this PR, I discovered that:
- Set the toggle Off (no auto-extract)
- Load a graph (observe nothing was extracted)
- Set the toggle On (observe nodes auto-extracted)

Then,
- Set the toggle Off again (no auto-extract)

Expected: nothing should be extracted
Actual: high degree nodes still extracted, even though the toggle is Off

I think this surprising behavior manifests because each node's `.include` property is not cleared upon changing the toggle. To correct this, could we split it this L179 observer into a new one separate from the rest ('graphHierarchy', 'edgeWidthFunction', ...), which resets each node's `.include` to be unspecified?

Something like:

```ts
@observe('autoExtractNodes')
_onAutoExtractNodesChanged() {
    if (!this.graphHierarchy) return;
    for (const node of Object.values(this.graphHierarchy.getNodeMap())) {
        node.include = tf_graph.InclusionType.UNSPECIFIED;
    }
    this._buildRenderHierarchy(this.graphHierarchy);
  }
```
?

Reply:
Ok. This will reset manually included/excluded nodes, but I agree that is better than the current behavior.